sol/main.c

```diff
@@ -690,7 +690,7 @@ run_sol(struct net_config *net_conf, struct sol_config *sol_conf)
 	 * Although the packets are going to the back interface,
 	 * they are allocated at the front interface.
 	 */
-	front_inc = 2 * sol_conf->pri_req_max_len + sol_conf->enq_burst_size;
+	front_inc = 2 * sol_conf->pri_req_max_len__ + sol_conf->enq_burst_size;
```

Comment:
We need to multiply this whole expression by the number of instances; see another comment for more information.